plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -1072,10 +1073,10 @@ public void testLike()
                 .isFullyPushedDown();
 
         assertThat(query("" +
-                 "SELECT " +
-                 "text_column " +
-                 "FROM " + indexName + " " +
-                 "WHERE text_column LIKE 's_.m%ex\\t'"))
+                "SELECT " +
+                "text_column " +
+                "FROM " + indexName + " " +
+                "WHERE text_column LIKE 's_.m%ex\\t'"))
```

Comment:
Can we revert the indentations ?

Reply:
I am not see any issue with indentation on my editor. Just compared with the previous version i don't see any change in the indentation. Can you please check once ?